pymc_bart/utils.py

```diff
@@ -57,7 +60,7 @@ def _sample_posterior(
         flatten_size *= s
 
     idx = rng.integers(0, len(stacked_trees), size=flatten_size)
-    shape = stacked_trees[0][0].predict(X[0]).size
+    shape = stacked_trees[0][0].predict(x=X[0], m=m).size
```

Comment:
This actually for a line below this

            p += np.vstack([tree.predict(x, excluded) for x in X])

should be
            
             p += np.vstack([tree.predict(x, m=m, excluded=excluded) for x in X])

Reply:
added in https://github.com/pymc-devs/pymc-bart/pull/79/commits/996596f2053bd4bb1b942a5b483d712d84b042c0